dsgrid/dimension/standard.py

```diff
@@ -122,19 +122,19 @@ class EnergyEfficiency(MetricDimensionBaseModel):
 # ---------------------------
 # TIME DIMENSIONS
 # ---------------------------
-class Time(TimeDimensionModel):
+class Time(DateTimeDimensionModel):
     """Time attributes"""
 
 
-class Timezone(TimeDimensionModel):
+class Timezone(DateTimeDimensionModel):
     """Timezone attributes"""
 
 
-class DayType(TimeDimensionModel):
+class DayType(DateTimeDimensionModel):
     """Day Type attributes"""
 
 
-class Season(TimeDimensionModel):
+class Season(DateTimeDimensionModel):
     """Season attributes"""
```

Comment:
I am confused as to why Timezone, DayType, and Season were deriving from TimeDimensionModel (and are now deriving from DateTimeDimensionModel)

Reply:
I performed a global swap of TimeDimensionModel to DateTimeDimensionModel. DayType and Season are actually duplicate of enums defined elsewhere. Timezone is unused. These don't appear to make sense with our current implementation. If we want to allow users to specify season as a time dimension then we'll need to make a new data model. We should probably delete these until someone needs one.